src/dosext/dpmi/dpmi.c

```diff
@@ -2278,7 +2278,7 @@ static void do_int31(sigcontext_t *scp)
 	  _LWORD(edx) = 0;
 	  *buf = DPMI_VERSION;
 	  *(buf+1) = DPMI_DRIVER_VERSION;
-	  sprintf(buf+2, "DOSEMU Version %d.%d\n", VERSION_NUM, SUBLEVEL);
+	  snprintf(buf+2, 126, "DOSEMU Version %d.%d", VERSION_NUM, SUBLEVEL);
```

Comment:
126??

Reply:
Seems to be the size of buffer remaining(128 - 2),  http://www.delorie.com/djgpp/doc/dpmi/api/310401.html